diff --color -Nur ansible-2.9.23.orig/docs/docsite/_extensions/pygments_lexer.py ansible-2.9.23/docs/docsite/_extensions/pygments_lexer.py
--- ansible-2.9.23.orig/docs/docsite/_extensions/pygments_lexer.py	2021-06-21 23:04:48.000000000 -0700
+++ ansible-2.9.23/docs/docsite/_extensions/pygments_lexer.py	2021-07-10 12:33:05.790899630 -0700
@@ -178,7 +178,7 @@
         See http://www.sphinx-doc.org/en/stable/extdev/index.html#dev-extensions.
     """
     for lexer in [
-        AnsibleOutputLexer(startinline=True)
+        AnsibleOutputLexer
     ]:
         app.add_lexer(lexer.name, lexer)
         for alias in lexer.aliases:
